app/_src/guides/transition-to-the-new-policies.md

```diff
@@ -0,0 +1,411 @@
+  conf:
+    connectTimeout: 21s
+    tcp: 
+      idleTimeout: 22s
+    http:
+      idleTimeout: 22s
+      requestTimeout: 23s
+      streamIdleTimeout: 25s
+      maxStreamDuration: 26s' | kubectl apply -f-
+```
+
+### Deploy CircuitBreaker
+
+```sh
+echo 'apiVersion: kuma.io/v1alpha1
+kind: CircuitBreaker
+mesh: default
+metadata:
+  name: cb-global
+spec:
+  sources:
+  - match:
+      kuma.io/service: "*"
+  destinations:
+  - match:
+      kuma.io/service: "*"
+  conf:
+    interval: 21s
+    baseEjectionTime: 22s
+    maxEjectionPercent: 23
+    splitExternalAndLocalErrors: false
+    thresholds:
+      maxConnections: 24
+      maxPendingRequests: 25
+      maxRequests: 26
+      maxRetries: 27
+    detectors:
+      totalErrors: 
+        consecutive: 28
+      gatewayErrors: 
+        consecutive: 29
+      localErrors: 
+        consecutive: 30
+      standardDeviation:
+        requestVolume: 31
+        minimumHosts: 32
+        factor: 1.33
+      failure:
+        requestVolume: 34
+        minimumHosts: 35
+        threshold: 36' | kubectl apply -f-
+```
+
+## Transition to the new policies
+
+Each type of policy is migrated separately; for example, once we have completely finished with the Timeouts, 
+we will proceed to the next policy type, CircuitBreakers.
+
+The generalized migration process roughly consists of 4 steps:
+
+1. Create and deploy the replacement policies in [shadow mode](/docs/{{ page.version }}/policies/applying-policies/#applying-policies-in-shadow-mode). 
+The corresponding new policy type can be found in [the table](/docs/{{ page.version }}/policies/introduction).
+2. Review the list of changes that're going to be created by the replacement policy.
+3. Remove `kuma.io/effect: shadow` label so that policy is applied in a normal mode.
+4. Observe metrics, traces and logs. If something goes wrong change policy's mode back to shadow and return to the step 2.
+If everything is fine then remove the old policies.
+
+### TrafficPermission -> MeshTrafficPermission
+
+1. Create a replacement policy for `app-to-redis` TrafficPermission and apply it with `kuma.io/effect: shadow` label:
+
+    ```sh
+    echo 'apiVersion: kuma.io/v1alpha1
+    kind: MeshTrafficPermission
+    metadata:
+      namespace: {{site.mesh_namespace}}
+      name: app-to-redis
+      labels:
+        kuma.io/mesh: default
+        kuma.io/effect: shadow
+    spec:
+      targetRef:
+        kind: MeshService
+        name: redis_kuma-demo_svc_6379
+      from:
+        - targetRef:
+            kind: MeshSubset
+            tags: 
+              kuma.io/service: demo-app_kuma-demo_svc_5000
+          default:
+            action: Allow' | kubectl apply -f -
+    ```
+
+2. Check the list of changes for `redis_kuma-demo_svc_6379` pod in Envoy configuration using `kumactl`, `jq` and `jd`:
+
+    ```sh
+    kumactl inspect dataplane redis-8fcbfc795-twlst.kuma-demo --type=config --shadow --include=diff | jq '.diff' | jd -t patch2jd
```

Comment:
can you separate output to different block, right now when you use copy button you will also copy output